torchrl/envs/common.py

```diff
@@ -2393,14 +2393,16 @@ def rollout(
                 the call to ``rollout``.
 
         Keyword Args:
-            auto_reset (bool, optional): if ``True``, resets automatically the environment
-                if it is in a done state when the rollout is initiated.
-                Default is ``True``.
```

Comment:
IMO this docstring was wrong and caused some confusion on our side. I hope that I was able to get the correct meaning of this argument.

Reply:
That looks great!